site/data/examples.yml

```diff
@@ -1,38 +1,46 @@
 - category: Starters
   external: true
-  description: "Functional examples of using Bootstrap in common JS frameworks like Webpack, Parcel, Vite, and more you can edit in StackBlitz."
```

Comment:
All these "Bootstrap" -> "OUDS Web" must be reverted. There are Bootstrap examples. We got a warning callout explaining how to modify them.